### ISSUE TITLE
Error: missing object 'tSQLt.Private_MarktSQLtTempObject' in examples.sql

### ISSUE THREAD

**Issue by ssg47** [tSQLt-org/tSQLt#154]
I've just installed v1.0.8043.39707 on a new server.
I've opened and run `PrepareServer.sql` in SSMS, selecting No to the message about inconsistent line endings.
I've opened and run `Example.sql`.
I received this message:
```
Installed at 2022-02-06 08:12:37.243
The module 'Private_NoTransactionHandleTable' depends on the missing object 'tSQLt.Private_MarktSQLtTempObject'. The module will still be created; however, it cannot run successfully until the object exists.
 
+-----------------------------------------+
|                                         |
| Thank you for using tSQLt.              |
|                                         |
| tSQLt Version: 1.0.8043.39707           |
|                                         |
+-----------------------------------------+

Completion time: 2022-02-06T08:12:38.3026463+00:00
```

### Version info:
```
SQL Server Management Studio				15.0.18390.0
SQL Server Management Objects (SMO)			16.100.46521.71
Microsoft Analysis Services Client Tools		15.0.19714.0
Microsoft Data Access Components (MDAC)			10.0.14393.0
Microsoft MSXML						3.0 6.0 
Microsoft .NET Framework				4.0.30319.42000
Operating System					10.0.14393
```

**Comment by ZekeLu** [comment]
I think this is a duplicate of #150, and has been fixed by #151.

And since the object `tSQLt.Private_MarktSQLtTempObject` has been created later, you can ignore  the warning message for now.

**Comment by lizbaron** [comment]
Per @ZekeLu's comment. This is a duplicate of #150. We plan to release a new version with this fix by 2022-02-14. We are closing this ticket.
